java8/src/main/java/Client/Main.java

```diff
@@ -0,0 +1,62 @@
+package Client;
+
+import Data.Car;
+
+import java.util.*;
+
+import static Client.LogicalOperations.*;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        //1
+        Car c1 = new Car("Mercedes", true, 4);
+        Car c2 = new Car("Mazda", true, 4);
+        Car c3 = new Car("Maseratti", false, 4);
+        Car c4 = new Car("Jaguar", true, 3);
+        Car c5 = new Car("Dacia", false, 5);
+        Car c6 = new Car("McLaren", false, 2);
+
+        List<Car> carsList = getObjects(c1, c2, c3, c4, c5, c6);
+        //2
+        HashSet<Car> set = filterCarsWithTriggers(carsList);
+        //3
+        findMinByName(set);
+        //4
+        HashSet<String> stringSet = new HashSet<>();
+        addRandomString(stringSet);

```

Comment:
you don't have to call this method five times, either add Strings based on a number (in that function) or create a loop